.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
```

Comment:
AFAIK the latest `recommended` now includes `eslint-recommended` and will also set the parser, sourceType and plugin

Reply:
Do you mean the latest `plugin:@typescript-eslint/recommended`?

Reply:
Yes sorry, just realised how unclear that was! 😂 They also publish a strict config now too but it needs type information

Reply:
Fixed!